docs/pom.xml

```diff
@@ -42,6 +42,12 @@
 					<plugin>
 						<groupId>org.asciidoctor</groupId>
 						<artifactId>asciidoctor-maven-plugin</artifactId>
+						<configuration>
+							<attributes>
+								<project-version>${project.version}</project-version>
```

Comment:
I intentionally kept the names slightly different (hyphens) so that future chris or oleg does not think that all the pom properties are magically available w/o us updating this file. Albeit the asciidocs syntax to sub the props is slightly different than maven's (`{ varName }` vs `${ varName }` respectively) but still, I don't want to scratch my head over this later.